python/gui/MaaS/templates/maas/dataset_management.html

```diff
@@ -0,0 +1,797 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>OWP MaaS</title>
+    {% load static %}
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.0/jquery.min.js">
+    </script>
+    <script src="https://code.jquery.com/ui/1.12.1/jquery-ui.js">
+    </script>
+    <script src="{% static 'maas/js/components/confirmDialog.js' %}">
+    </script>
+    <script src="{% static 'maas/js/components/createDatasetForm.js' %}">
+    </script>
+    <script src="{% static 'maas/js/components/datasetOverview.js' %}">
+    </script>
+    <link rel="stylesheet" type="text/css" href="https://ajax.googleapis.com/ajax/libs/jqueryui/1.12.1/themes/cupertino/jquery-ui.css">
+    <link rel="icon" type="image/gif" href="{% static 'maas/img/favicon.gif' %}">
+    <script>
+        function activate_model(model) {
+            $(".parameters").hide();
+            $("#" + model + "_parameters").show();
+        }
+
+        function buildDatasetsOverviewTable() {
+            $.ajax({
```

Comment:
In the future, we should avoid bringing in `jquery`. With the advent of the [`fetch`](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch) API, it has become the preferred way to make requests.